tox.ini

```diff
@@ -148,7 +148,7 @@ description = Run scenario tests
 deps =
     -r{toxinidir}/requirements.txt
     pytest
-    ops-scenario
+    ops-scenario == 3.0.2
```

Comment:
If we are pinning an exact version, we should have a comment explaining why and when we can stop.

Reply:
To pin or not to pin... that's the question @PietroPasotti ;-)